modules/33acrossBidAdapter.js

```diff
@@ -85,6 +83,26 @@ function _validateVideo(bid) {
     return false;
   }
 
+  const placement = videoAdUnit.placement || videoBidderParams.placement
+
+  // If placement if defined, it must be a number
+  if (
+    typeof placement !== 'undefined' &&
+    typeof placement !== 'number'
+  ) {
+    return false;
+  }
+
+  const startdelay = videoAdUnit.startdelay || videoBidderParams.startdelay
+
+  // If startdelay is defined it must be a number
+  if (
```

Comment:
for `startdelay`, do we need to check the product === instream?

Reply:
Oh meaning if `outstream` don't even bother validating it if it's set?

Reply:
correct, in case to increase a little bit traffic

Reply:
Done. See following line.